### PR TITLE
Product Test

### LINKED ISSUE #218: Product Test

### PR DESCRIPTION
**Pull request by jicsontoh**
Testing for 
- Help Command
- Sort Command
- Edit Command

1. Updated help command to not change UI after using
2. Edited some command messages
3. Fixed the spaces in input for Sort Command
4. Updated Remark in UG
5. Update Edit Command error messages
6. Update Error Message for Sort, Buy, Sell

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -284,7 +284,7 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleFilterTransaction(commandResult);
             } else if (commandResult.isSortedTransactions()) {
                 handleSortTransaction(commandResult);
-            } else {
+            } else if (!commandResult.hasNoUiChange()) {
                 handleClientDetailsUpdate(commandResult);
             }
```
Comment (Jo3LW) [inline src/main/java/seedu/address/ui/MainWindow.java:293]: Great Code! Maybe adding a default case would improve code quality :)

**Review by Jo3LW — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (jicsontoh) [reply src/main/java/seedu/address/ui/MainWindow.java:293]: Good idea! Thanks!

**Review by notle1706 — APPROVED** [review]
LGTM!

**Pull request merged**